u8timeseries/models/fft.py

```diff
@@ -54,30 +48,30 @@ def crop_to_match_seasons(series: 'TimeSeries', required_matches: set):
 
 class FFT(AutoRegressiveModel):
 
-    def __init__(self, filter_first_n: int = -1, required_matches: set = {}, detrend: bool = False, detrend_poly_degree: int = 3):
+    def __init__(self, nr_freqs_to_keep: Optional[int] = None, required_matches: set = {}, trend: bool = False, trend_poly_degree: int = 3):
```

Comment:
about the `{}` default value: you shouldn't have mutable defaults. It can lead to nasty behaviors (see e.g. this: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)

Reply:
Wow, I would not have expected that behavior of default arguments at all. Thanks for making me aware of this!